.github/workflows/trivy_scan.yml

```diff
@@ -0,0 +1,29 @@
+name: Trivy vulnerability scan
+on:
+  workflow_dispatch:
+  schedule:
+  - cron: '0 4 * * *' # Nightly 4AM UTC
+jobs:
+  build:
+    name: Trivy
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy vulnerability scanner in repo mode
+        uses: aquasecurity/trivy-action@master
+        with:
+          scan-type: 'fs'
+          ignore-unfixed: true
+          format: 'sarif'
+          output: 'trivy-results.sarif'
+          severity: 'CRITICAL'
+          skip-dirs: 'website/,tools/,infrastructure/,test/,orbit/pkg/insecure/'
+          trivyignores: '.trivyignore'
+          security-checks: 'vuln'
+
+      - name: Upload Trivy scan results to GitHub Security tab
```

Comment:
once is uploaded there, do we get any kind of notification? or is this tab actively monitored?

Reply:
For now we do not have notifications, but I am planning on having a recurring process to check it out + also to notify in Slack.

Reply:
sounds good! 👍